### DISCUSSION TITLE
Welcome, Taky0nDon!

### DISCUSSION THREAD

**Discussion by Taky0nDon** [ParanoidUser/yolo#459] [Subscribe]
I have so much to learn about how to use git but I'm grateful for what you are doing here @ParanoidUser

**Comment by ParanoidUser** [comment]
Ahoy @Taky0nDon, welcome aboard!

I'm very happy that you've found this project useful, and I hope that this community will help you expand your knowledge quickly. The invitation to join the collaborator group has been sent to you. Don't hesitate to ask any questions you may have along your GitHub journey. 😊
